core/services/ocr2/plugins/ocr2keeper/evm21/encoding/packer.go

```diff
@@ -162,6 +161,32 @@ func (p *abiPacker) UnpackReport(raw []byte) (automation_utils_2_1.KeeperRegistr
 	return report, nil
 }
 
+type StreamsLookupError struct {
+	FeedParamKey string
+	Feeds        []string
+	TimeParamKey string
+	Time         *big.Int
+	ExtraData    []byte
+}
+
+// DecodeStreamsLookupErr decodes the revert error StreamsLookup(string feedParamKey, string[] feeds, string feedParamKey, uint256 time, byte[] extraData)
+func (p *abiPacker) DecodeStreamsLookupErr(data []byte) (*StreamsLookupError, error) {
```

Comment:
nit. can we call it streamsLookupRequest? (it's via revert error but slightly confusing to call it error)